crates/bindings-csharp/Runtime/LogStopwatch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpacetimeDB
+{
+    public class LogStopwatch
+    {
+        private uint StopwatchId;
+
+        public LogStopwatch(string name)
+        {
+            StopwatchId = StartStopwatchInternal(name);
+        }
+
+        private static uint StartStopwatchInternal(string name)
+        {
+            var name_bytes = Encoding.UTF8.GetBytes(name);
+            var id = Internal.FFI._log_stopwatch_start(name_bytes, (uint)name_bytes.Length);
+            return id;
+        }
```

Comment:
This can be just inlined into the constructor, no need for extra indirection and should be easier to read.

```suggestion
        public LogStopwatch(string name)
        {
            var name_bytes = Encoding.UTF8.GetBytes(name);
            StopwatchId = Internal.FFI._log_stopwatch_start(name_bytes, (uint)name_bytes.Length);
        }
```

Reply:
Oh, guess it's leftover from when we still had `LogTimer.Start()`. Fixed